pysys/process/user.py

```diff
@@ -1165,21 +1165,27 @@ def mkdir(self, path):
 		@param path: The path to be created. This can be an absolute path or 
 		relative to the testcase output directory.
 		
-		@return: the same path passed, to facilitate fluent-style method calling. 
+		@return: the absolutized path of the new directory, to facilitate fluent-style method calling. 
```

Comment:
"absolutized" is not a word, I think "absolute" works just fine here.  Actually you could even say "...absolute path to..." rather than "...absolute path of..." which might be a bit nicer gramatically.